src/providers/ses.rs

```diff
@@ -0,0 +1,89 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
There are no tests for this SES functionality yet, because I haven't worked out how to mock all Rusoto stuff. Hopefully it's okay to get round to that later, but in the meantime you should be able to find an email in your inbox with the subject "fxa-email-service test" that shows it works.

Reply:
> you should be able to find an email in your inbox with the subject "fxa-email-service test" that shows it works.

Can confirm :-)